### DISCUSSION TITLE
Debian 12 (bookworm) new Kernel, sound broken?

### DISCUSSION THREAD

**Discussion by unosonic** [tphakala/birdnet-go#116] [General]
hi, I've installed birdnet-go (non-docker) on an fresh RaspiOS (Debian12 based), everything has worked until recent update/upgrade, which introduced a new Linux kernel (6.6.20), before I had 6.1.x. This upgrade killed any sound on the system. Soundcard shows up, but only show I/O errors. There's widespread complaints on this on the net, yet inconsistent. Does anyone have made similar observations? After heavy debugging. I've downgraded fo "legacy" RaspiOs, i.e. Debian 11.

**Comment by tphakala** [comment]
I am not seeing any issues. My BirdNET-Go development system is RPI 5 running Bookworm, often with latest updates.,

This is kernel version I am running at the moment
"Linux buildpi5 6.6.20+rpt-rpi-2712 #1 SMP PREEMPT Debian 1:6.6.20-1+rpt1 (2024-03-07) aarch64 GNU/Linux"

**Reply by unosonic** [reply]
good to hear, I've only tested on a pi zero (not w/ birdnet-go) and a pi3 (arecord, darkice, birdnet-go), both went silent... will check newer updates
